README.md

```diff
@@ -1 +1,222 @@
-# cisco-securex-orchestration-workflows
\ No newline at end of file
+# Cisco SecureX Integration
```

Comment:
Can the repo name be just SecureX , similar to : https://github.com/Tufin/SecureX

Reply:
@anvesh-cohesity  , we do the folder have subfolders ? should just be atomics/*.json and workflows/*.json ?

Reply:
SecureX is broad name, it holds a lot of applications here. here we are mostly integrating with orchestration piece in SecureX, in future if we have to build an integration using relay server and have secureX tile dashboards.. or with amp endpoints , all these cannot go into a single repo SecureX. trying to keep them separate. 

and yes we have Atomics and Workflows folders with with json files in it. similar to Cisco's repo https://github.com/CiscoSecurity/sxo-05-security-workflows